manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml

```diff
@@ -58,17 +58,17 @@ spec:
         - name: MYSQL_DATABASE
           valueFrom:
             configMapKeyRef:
-              name: metadata-configmap
+              name: metadata-mysql-configmap
```

Comment:
Let's keep this around and add a new config map to maintain backward compatibility.

Reply:
Done.